apps/admin-gui/src/assets/i18n/en.json

```
@@ -1648,7 +1650,8 @@
       "GENERATE": "Create",
       "SUCCESS": "Members successfully created",
       "EMAIL": "Email",
-      "EMAIL_ERROR": "Email is not valid"
+      "EMAIL_ERROR": "Email is not valid",
+      "PASSWORD_RESET": "Send activation link to entered email"
```

```suggestion
      "PASSWORD_RESET": "Let created users change their password via given email addresses."
```
@xkureck what do you think of this suggestion?

Also, @bodnara, you should probably use different values in the two dialogs (creating one user vs multiple users)